hydrolysis-analyzer.html

```diff
@@ -93,14 +101,22 @@
       }
       this._setLoading(true);
 
+      var baseUri = this.ownerDocument.baseURI;
+      var additionalImports;
```

Comment:
can you document `additionalImports`? The name sounds to me like additional files that are use for analysis, but maybe not part of the primary analysis targets themselves. While `sources` sounds like it's the set of entrypoint to start analyzing from. This difference in names make me look for differences in the behavior...